administrator/components/com_patchtester/script.php

```diff
@@ -48,6 +51,24 @@ public function __construct()
 	 */
 	public function postflight($type, $parent)
 	{
-		$this->removeFiles();
+		if ($type == 'install')
+		{
+			$language = JFactory::getLanguage();
```

Comment:
I am not sure, do we need to load the language files? I thought the installer automatically loads the sys.ini language file.

Reply:
I'm personally unsure, I didn't see it work without the line if memory serves me correctly.

Reply:
We can test and see what happens.